libraries/AP_HAL/utility/packetise.h

```diff
@@ -1,4 +1,10 @@
 #pragma once
+#include <AP_HAL/AP_HAL.h>
+#include <GCS_MAVLink/GCS_config.h>
+
+#ifndef AP_MAVLINK_PACKETISE_ENABLE
```

Comment:
```suggestion
#ifndef AP_MAVLINK_PACKETISE_ENABLED
```

It's a niggle but also what we're trying to standardise on

... also, do you still need `HAL.h`?

Reply:
Fixed